gnucash/import-export/import-main-matcher.c

```diff
@@ -553,9 +565,18 @@ on_matcher_ok_clicked (GtkButton *button, GNCImportMainMatcher *info)
         if (first_tran)
         {
             Split* first_split = gnc_import_TransInfo_get_fsplit (trans_info);
-            xaccAccountSetAppendText (xaccSplitGetAccount(first_split), append_text);
+            Account* account = xaccSplitGetAccount (first_split);
+            xaccAccountSetAppendText (account, append_text);
             first_tran = false;
+            begin_edit (&accounts_modified, account);
         }
+
+        if (gnc_import_TransInfo_get_action (trans_info) == GNCImport_ADD)
```

Comment:
Why do you only do this for added transactions and not for updated transactions ? Is there no change to the account while updating ? Amounts may be updated ?